runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,294 @@
+					prayerHealValue, PRAYER_HEAL_COLOR);
+		}
+
+		if (config.enableSkillIcon() || config.enableCounter())
+		{
+			renderIconsAndCounters(g, offsetPrayerX, offsetPrayerY, prayerImage.getScaledInstance(IMAGE_SIZE, IMAGE_SIZE, Image.SCALE_SMOOTH), counterPrayerText, PRAYER_LOCATION_X);
+			renderIconsAndCounters(g, offsetHealthX, offsetHealthY, healthImage, counterHealthText, HEALTH_LOCATION_X);
+		}
+
+		return null;
+	}
+
+	private void renderBar (Graphics2D graphics, int x, int y, int max, int current, int height, Color filled)
+	{
+		graphics.setColor(BACKGROUND);
+		graphics.drawRect(x, y, WIDTH - PADDING, height - PADDING);
+		graphics.fillRect(x, y, WIDTH, height);
+
+		int filledHeight = getBarHeight(max, current, height);
+		graphics.setColor(filled);
+		graphics.fillRect(x + PADDING,
+				y + PADDING + (height - filledHeight),
+				WIDTH - PADDING * OFFSET,
+				filledHeight - PADDING * OFFSET);
+	}
+
+	private void renderHealingBar(Graphics2D graphics, int x, int y, int max, int current, int height, int heal, Color color)
+	{
+		int filledHeight = getBarHeight(max, heal, height);
+		int filledCurrentHeight = getBarHeight(max, current, height);
+		int overHeal;
+		graphics.setColor(color);
+
+		if (filledHeight + filledCurrentHeight > height)
+		{
+			overHeal = filledHeight + filledCurrentHeight - height;
+			filledHeight = filledHeight - overHeal + OVERHEAL_OFFSET;
+			graphics.setColor(OVERHEAL_COLOR);
+			graphics.fillRect(x + PADDING,
+					y - filledCurrentHeight + (height - filledHeight) + HEAL_OFFSET,
+					WIDTH - PADDING * OVERHEAL_OFFSET,
+					filledHeight - PADDING * OVERHEAL_OFFSET);
+		}
+		else
+		{
+			graphics.fillRect(x + PADDING,
+					y - OVERHEAL_OFFSET - filledCurrentHeight + (height - filledHeight) + HEAL_OFFSET,
+					WIDTH - PADDING * OVERHEAL_OFFSET,
+					filledHeight + OVERHEAL_OFFSET - PADDING * OVERHEAL_OFFSET);
+		}
+	}
+
+	private void renderIconsAndCounters(Graphics2D graphics, int x, int y, Image image, String counterText, int counterPadding)
+	{
+		int widthOfCounter = graphics.getFontMetrics().stringWidth(counterText);
+		int centerText = (WIDTH - PADDING) / 2 - (widthOfCounter / 2);
+
+		if (config.enableCounter())
+		{
+			graphics.setFont(FontManager.getRunescapeSmallFont());
+			textComponent.setColor(Color.WHITE);
+			textComponent.setText(counterText);
+			textComponent.setPosition(new java.awt.Point(x + centerText + counterPadding, y + COUNTER_ICON_HEIGHT));
+		}
+		else
+		{
+			textComponent.setText("");
+		}
+
+		if (config.enableSkillIcon())
+		{
+			graphics.drawImage(image, x + ICON_AND_COUNTER_OFFSET_X + PADDING, y + ICON_AND_COUNTER_OFFSET_Y - image.getWidth(null), null);
+			textComponent.setPosition(new java.awt.Point(x + centerText + counterPadding, y + SKILL_ICON_HEIGHT));
+		}
+
+		textComponent.render(graphics);
+	}
+
+	private static int getBarHeight ( int base, int current, int size)
+	{
+		double ratio = (double) current / (double) base;
```

Comment:
Only one of the ints needs to be typecast

```suggestion
		double ratio = (double) current / base;
```